docs/USER_MANUAL.md

```
@@ -0,0 +1,1943 @@
+The viewer and deception summary parsers also accept the hidden compatibility
+option `--paired`. It has the same effect as `--dual-path`.
+
+### 24.6 RL commands
+
+| Command | Required options | Optional options |
+| --- | --- | --- |
+| `gepa rl train` | `--config` | `--backend`; `--endpoint`; `--learner`; `--coordinator-command`; `--actor-endpoint`; `--max-steps`. |
+| `gepa rl resume` | `--config`, `--checkpoint` | The common backend options and `--max-steps`. |
+| `gepa rl collect` | `--config` | The common backend options; `--dataset`; `--output`. |
+| `gepa rl evaluate` | `--config` | The common backend options. |
+| `gepa rl doctor` | None | `--config`; `--backend system|llama-cpp-vulkan`; `--endpoint`. |
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)